templates/extras/calc.md

```diff
@@ -86,31 +53,37 @@ the operators, but actually `calc` will work with any relation for which
 the corresponding transitivity statement is tagged `[trans]`:
```

Comment:
This is outdated. This attribute no longer exists.

Reply:
Whoops sorry missed that one